consultation_analyser/support_console/jinja2/support_console/consultations/import_consultation.html

```diff
@@ -0,0 +1,170 @@
+            <strong>Missing file:</strong> {{ error.message }}
+          {% elif error.type == 'missing_output' %}
+            <strong>Missing AI output:</strong> {{ error.message }}
+          {% elif error.type in ['invalid_format', 'empty_file'] %}
+            <strong>File format issue:</strong> {{ error.message }}
+          {% elif error.type == 'no_questions' %}
+            <strong>Structure issue:</strong> {{ error.message }}
+          {% elif error.type == 'file_not_found' %}
+            <strong>File not found:</strong> {{ error.message }}
+          {% elif error.type == 'system_error' %}
+            <strong>System error:</strong> {{ error.message }}
+          {% else %}
+            {{ error.message }}
+          {% endif %}
+        </li>
+        {% endfor %}
+      </ul>
+      
+      {% set missing_files = validation_errors | selectattr('type', 'equalto', 'missing_file') | list %}
+      {% set missing_outputs = validation_errors | selectattr('type', 'equalto', 'missing_output') | list %}
+      {% set format_issues = validation_errors | selectattr('type', 'in', ['invalid_format', 'empty_file']) | list %}
+      
+      {% if missing_files or missing_outputs %}
+      <div class="govuk-inset-text">
+        <h3 class="govuk-heading-s">What you need to do:</h3>
+        <ul class="govuk-list govuk-list--bullet">
+          {% if missing_files %}
+          <li>Upload the missing consultation files to your S3 bucket</li>
+          {% endif %}
+          {% if missing_outputs %}
+          <li>Run the AI analysis pipeline to generate the missing output files</li>
+          {% endif %}
+          {% if format_issues %}
+          <li>Check that your JSON/JSONL files are properly formatted</li>
+          {% endif %}
+        </ul>
+        <p class="govuk-body-s">Once you've resolved these issues, you can try importing again.</p>
+      </div>
+      {% endif %}
+      
+      <details class="govuk-details">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">
+            Show technical details
+          </span>
+        </summary>
+        <div class="govuk-details__text">
+          <ul class="govuk-list">
+            {% for error in validation_errors %}
+            <li><code>{{ error.technical }}</code></li>
+            {% endfor %}
+          </ul>
+        </div>
+      </details>
+    </div>
+  </div>
+</div>
+{% endif %}
+
+<p class="govuk-body-l">Import complete consultation data including AI analysis in one operation.</p>
+<p class="govuk-body">This will import both the consultation data (questions, respondents, responses) and AI analysis
+  results (themes, sentiment, evidence) sequentially.</p>
+
+<p class="govuk-body">Data should be saved in: <code>{{ bucket_name }}/app_data/[CONSULTATION FOLDER]</code></p>
+
+{{ govukDetails({
+'summaryText': 'Expected S3 Structure',
+'html': '
+<pre><code>&lt;consultation-folder&gt;/
```

Comment:
nit: Any idea how to get rid of:
![image](https://github.com/user-attachments/assets/62c68e93-1e37-4ada-b3d6-a8ed93919560)

Nice having that info in the expander.